RULES.md

```diff
@@ -13,350 +13,441 @@ Notices are split into three categories: `INFO`, `WARNING`, `ERROR`.
+| [`MissingRequiredFieldError`](#MissingRequiredFieldError)                                                       	| A required field is missing.                                                                                                                                	| `missing_required_field`                                	|
+| [`MissingRequiredFileError`](#MissingRequiredFileError)                                                         	| A required file is missing.                                                                                                                                 	| `missing_required_file`                                 	|
+| [`MissingTripEdgeNotice`](#MissingTripEdgeNotice)                                                               	| Missing trip edge `arrival_time` or `departure_time`.                                                                                                       	| `missing_trip_edge_arrival_time_departure_time`         	|
+| [`NewLineInValueNotice`](#NewLineInValueNotice)                                                                  	| New line or carriage return in a value in CSV file.                                                                                                         	| `new_line_in_value`                                     	|
+| [`NumberOutOfRangeError`](#NumberOutOfRangeError)                                                               	| Out of range value.                                                                                                                                         	| `number_out_of_range`                                   	|
+| [`OverlappingFrequencyNotice`](#OverlappingFrequencyNotice)                                                     	| Trip frequencies overlap.                                                                                                                                   	| `overlapping_frequency`                                 	|
+| [`RouteBothShortAndLongNameMissingNotice`](#RouteBothShortAndLongNameMissingNotice)                             	| Missing route short name and long name.                                                                                                                     	| `route_both_short_and_long_name_missing`                	|
+| [`SameNameAndDescriptionForRouteNotice`](#SameNameAndDescriptionForRouteNotice)                                 	| Same name and description for route.                                                                                                                        	| `same_route_name_and_description`                       	|
```

Comment:
> We should review all rule class names and codes to make sure they follow the same naming pattern (this could be done in another PR). They should be the same, but there are a few that are different like this example - SameNameAndDescriptionForRouteNotice and same_route_name_and_description.

Seehttps://github.com/MobilityData/gtfs-validator/pull/783

Reply:
> Also, I'd suggest we put the entity name first when it makes sense for consistency, and because if we list the rules in the table alphabetically then, for example, most of the Route rules should be grouped together. So, for example, this class and code could be renamed to RouteSameNameAndDescriptionNotice and route_same_name_and_description. Then it would appear right after RouteBothShortAndLongNameMissingNotice.

👍

Reply:
> Also, I'd suggest we put the entity name first when it makes sense for consistency, and because if we list the rules in the table alphabetically then, for example, most of the Route rules should be grouped together. So, for example, this class and code could be renamed to RouteSameNameAndDescriptionNotice and route_same_name_and_description. Then it would appear right after RouteBothShortAndLongNameMissingNotice.

Will be done in https://github.com/MobilityData/gtfs-validator/pull/783